python/lsst/ip/isr/isrTask.py

```diff
@@ -1590,6 +1596,34 @@ def run(self, ccdExposure, *, camera=None, bias=None, linearizer=None,
                                    amp.getName(), qaStats.getValue(afwMath.MEDIAN),
                                    qaStats.getValue(afwMath.STDEVCLIP))
 
+        # Calculate standard image quality statistics
+        if self.config.doStandardStatistics:
+            metadata = ccdExposure.getMetadata()
+            for amp in ccd:
+                ampExposure = ccdExposure.Factory(ccdExposure, amp.getBBox())
+                ampName = amp.getName()
+                metadata[f"LSST ISR MASK SAT {ampName}"] = isrFunctions.countMaskedPixels(
```

Comment:
Is prefixing these by `LSST` needed/desirable?

Reply:
Namespacing FITS HIERARCH headers with a observatory prefix is a convention to guarantee that you can't clash with anyone else. I don't see much of a downside in the "LSST" usage.